content/en/docs/migration/ot_overview.md

```diff
@@ -0,0 +1,121 @@
+OpenTelemetry Python API supports Python 3.5 and higher.
+
+## Migration Overview
+
+Currently, there are many codebases which are instrumented with OpenTracing. These
+codebases may use the OpenTracing API to instrument their apllication code, and they may
+install OpenTracing plugins to instrument the libraries and frameworks.
+
+For a simple approach to migrating to OpenTelemetry, you can do the following.
+
+1. Install the OpenTelemetry SDK, and remove the current OpenTracing implementation (e.g. the Jaeger client).
+2. Install the OpenTelemetry instrumentation libraries, and remove the OpenTracing equivalents.
+3. Update your dashboards, alerts, etc, to consume the new OpenTelemetry data.
+4. When writing new application code, write all new instrumentation using the OpenTelemetry API.
+5. Reinstrument your existing application code using the OpenTelemetry API when it is convenient.
+There is no hard requirement to remove existing OpenTracing API calls from your application, they will continue to work.
+
+The above approach works. However, replacing all existing OpenTracing instrumentation libraries at once could be burdensome. 
+To avoid any break in observability, we recommend that OpenTracing users progressively migrate their applications.
+The following migration guide presents a careful, incremental approach to transitioning to OpenTelemetry.
+
+### Step 1: install the OpenTelemetry SDK
+
+Before changing any instrumentation, ensure that you can switch to the OpenTelemetry SDK without causing
+any break in the telemetry the application currently emits. Doing this step on its own – without
+simultaneously introducing any new instrumentation – is recommended, as it makes it easier to determine
+whether there is any kind of break in instrumentation.
+
+1. Replace the OpenTracing Trace implementation you are currently using with the OpenTelemetry
+SDK. For example, if you are using the Jaeger, remove the Jaeger client and install the equivalent
+OpenTelemetry client.
+2. Install the OpenTracing Shim. This shim allows the OpenTelemetry SDK to consume OpenTracing instrumentation.
+3. Configure the OpenTelemetry SDK to export data using the same protocol and format that 
+the OpenTracing client was using. For example, if you were using an OpenTracing client that exported 
+tracing data in Zipkin format, configure the OpenTelemetry client to do the same.
+4. Alternatively, configure the OpenTelemetry SDK to emit OTLP, and send the data to a Collector, where 
+you can manage exporting data in multiple formats.
+
+Once you have the OpenTelemetry SDK installed, confirm that you can deploy your application and still
+recieve the same OpenTracing-based telemetry. In other words, corfirm that your dashboards, alerts, and other
+tracing-based analysis tools are still working.
+
+### Step 2: progressively replace instrumentation
+
+Once the OpenTelemetry SDK is installed, all new instrumentation can now be written using the OpenTelemetry API. With few
```

Comment:
Nit: if you could line-wrap the paragraphs in this section too, that would be great.

Reply:
Done, line wrapped everything at 80